### ISSUE TITLE
Accessibility - ARIA IDs and Labels

### ISSUE THREAD

**Issue by tander08** [QualitativeDataRepository/dataverse#74]
## Formatting Data: https://qdr.syr.edu/guidance/managing/formatting-data
(And all webpages that utilize this menu)

- [ ] 
![image](https://user-images.githubusercontent.com/89411909/132021016-bca47714-afcd-4c15-aae2-6d728ec4521f.png)
![image](https://user-images.githubusercontent.com/89411909/132023195-ef4156be-4aec-4bc3-8c72-3aeefbd83456.png)

## Data Project Types: https://qdr.syr.edu/deposit/projects
(And all webpages that utilize this menu)
- [ ] 
![image](https://user-images.githubusercontent.com/89411909/132531423-e00df259-ba39-4e9d-96b3-1b63940f210b.png)
![image](https://user-images.githubusercontent.com/89411909/132531596-aa68c1f0-a996-4991-840c-b8c16ae3e997.png)

## Annotation for Transparent Inquiry (ATI) at a Glance: https://qdr.syr.edu/ati
(And all webpages that utilize this menu)

![image](https://user-images.githubusercontent.com/89411909/132568144-7165d23a-0c2b-470b-83cd-639914f914d7.png)
![image](https://user-images.githubusercontent.com/89411909/132568175-1701d6bd-2a4b-489b-98f4-6e8abc5fdd9b.png)

## QDR Collection Development and Appraisal: https://qdr.syr.edu/policies/collectionsdevelopment
(And all webpages that utilize this menu)
![image](https://user-images.githubusercontent.com/89411909/132750550-b9dfa5eb-2266-424f-9bcc-274c62b42bd0.png)
![image](https://user-images.githubusercontent.com/89411909/132750586-b5bc3798-0375-4dd0-bda9-cac91579c61c.png)

## QDR Governance: https://qdr.syr.edu/about/governance
(And all webpages that utilize this menu)
![image](https://user-images.githubusercontent.com/89411909/132760262-2f3b52c9-e8cc-4d14-80d3-5327bbf63ba8.png)
![image](https://user-images.githubusercontent.com/89411909/132760292-d958b436-1cdf-4157-8241-f2453f4b5e1c.png)

## QDR Blog Pages (ex: https://qdr.syr.edu/qdr-blog/primr-and-qdr-help-irb-professionals-discuss-data-sharing-social-behavioral-and-economic)
(And all webpages that utilize this menu)
![image](https://user-images.githubusercontent.com/89411909/133294768-e9549636-f4c8-48e3-afa8-cc49eb51a6e3.png)

**Comment by qqmyers** [comment]
multiple
<h2 class="visually-hidden" id="block-mainmenu-menu">Main Menu Left Column</h2>
in the menus - same id

**Comment by qqmyers** [comment]
FWIW: I think the root of this issue is that the main menu is being included in the page twice:
left column - shown at full size
between title and content - shown when the screen width is reduced

The standard templating adds element ids based on the block name so, even though only one copy is shown at a time, there are two copies in the html. 

Looking at how best to fix it.

**Comment by qqmyers** [comment]
Looks like a reasonable way to fix this is to 
* create a new region in the theme (respmenu="Responsive Menu") and add the main menu to that as well as to the left column region (creating the region is in the code, assigning a copy of the main menu to it is a UI operation in the structure/blocks page.).
* add the page.respmenu element into the page instead of a second copy of the page.left_column (this is in code - the page.html.twig template)
* adjust css to avoid using the id for the region (a css change).

The overall effect here is that the initially hidden menu under the title has an id ending in mainmenu_2 which avoids duplicates.  Using the same menu in both regions avoids having a second copy of the menu itself which would have to stay in sync. The css change to use classes instead of the id (.block-menu.menu--main) is probably good practice anyway (in fact, it looks to me like the functionality to initially collapse the parts of the menu tree that aren't in the current path are broken in the responsive menu now/on prod because there are different css rules being applied to the different copies right now (a css path issue rather than the use of the #block-mainmenu id, but a demonstration of having two sets of rules that should have been kept in sync) - making both copies leverage the same rules as we need to do for this issue should also fix that.)

**Comment by adam3smith** [comment]
This looks like it's fixed? DId you go with the above?

**Comment by qqmyers** [comment]
Yes - in https://github.com/QualitativeDataRepository/drupal8/commit/4a2cabe4f0384eca38d80978cbfe0abeccc50975 (had to look it up!)
